docs/integrations/destinations/bigquery.md

```diff
@@ -135,6 +135,7 @@ Now that you have set up the BigQuery destination connector, check out the follo
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                  |
 | :------ | :--------- | :--------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------- |
+| 1.3.0   | 2023-04-18 | [25287](https://github.com/airbytehq/airbyte/pull/25287)   | Add configurable parameter for number of file buffers                                                                                                       |
```

Comment:
Minor nit: maybe add `#` in front of the hypertext, seems it was broken from the last merge